### DISCUSSION TITLE
Rioxarray reprojecting a dataset

### DISCUSSION THREAD

**Discussion by Iamnyatichi** [giswqs/geog-312#141] [Q&A] [answered]
Hello sir, I am encountering the following error, and I suspect is a GDAL installation causing it. How can I fix the GDAL installation on my computer? I have tried updating it ( conda update rasterio gdal ) but the error still persists.
![image](https://github.com/user-attachments/assets/9b33e15f-158b-44a7-96f5-1981f9168759)

**Comment by giswqs** [comment]
Yes, this a gdal installation issue on your computer. 

Try install gdal from the official conda channel. 

```python
conda create -n test python=3.12
conda activate test
conda install gdal
conda install -c conda-forge leafmap maplibre jupyterlab
```

**Reply by Iamnyatichi** [reply]
Hello sir, Thank you for help. However I am getting a new error as attached below. Kindly help
![image](https://github.com/user-attachments/assets/607e889d-fc8e-4706-ad83-b660da0a3fb7)

**Comment by giswqs** [comment, marked as answer]
See https://github.com/rasterio/rasterio/discussions/2721
